addons/web/static/src/legacy/action_adapters.js

```diff
@@ -227,7 +227,8 @@ function useMagicLegacyReload() {
             const controllerReload = widget.reload;
             widget.reload = function (...args) {
                 manualReload = true;
-                legacyReloadProm = controllerReload.call(widget, ...args);
+                legacyReloadProm = controllerReload.call(widget, ...args)
+                    .catch(() => legacyReloadProm = null);
                 return legacyReloadProm.then(() => {
```

Comment:
maybe simply replace .then by .always here. Because with your solution, you're also catching errors, which might hide issues.

Reply:
Done, I've also updated the commit message